CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ These changes are available on the `master` branch, but have not yet been releas
   `stage_speaker`, `stage_raise_hand`, `stage_topic`, and
   `guild_application_premium_subscription`.
   ([#1852](https://github.com/Pycord-Development/pycord/pull/1852))
+- Added new `EmbeddedActivity`s.
+  ([#1859](https://github.com/Pycord-Development/pycord/pull/1859))
```

Comment:
Since "EmbeddedActivitys" looks kinda funny, here's a quick fix.
```suggestion
- Added new `EmbeddedActivity` values. ([#1859](https://github.com/Pycord-Development/pycord/pull/1859))
```